CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+### 2.1 
+ * **IMPORTANT UPGRADE NOTE FOR CRUNCHY PACKAGE USERS**: In version 2.0, the Crunchy provided extras for node_exporter were split out from the pg##-extras package. A dependency was kept between these packages to make upgrading easier. For 2.1, the dependency between these packages has been removed. When upgrading from 1.7 or less, if you have node_exporter and postgres_exporter running on the same systems, ensure that you install the separate node_exporters_extras package after the update. See the README for the full package name(s).
+
+ * Minium required versions of software used in pgmonitor have been updated to: Prometheus 2.5.0, Prometheus Alertmanager 0.15.3, postgres_exporter 0.4.7 (enables full PG11 support) & Grafana 5.3.4.
+ * Fixed Grafana data source to use the "proxy" mode instead of "direct" with default install. Should fix issues encountered with the default setup when Grafana is not installed on the same system as Prometheus.
+ * Renamed functions_pg##.sql file to setup_pg##.sql to better clarify what it's for (and because it's not just functions).
```

Comment:
The way our setup works, the function creation requires that the users be created first so that privileges are set properly. We could document that one must be run before the other, but probably better for user-friendliness that it just exist in one file.

Reply:
User creation will only executed during install where as function may get updated with new releases. Having both in single file will generate unnecessary errors message or failure.

Reply:
Well, as of 10+, there hasn't been a need for functions anymore. So the setup is basically user creation right now anyway. If we end up needing any functions in the future, we can split it up then.

Reply:
We are changing functions "Note that for PG96 and lower, a new security definer function must be added (can just run setup_pg##.sql again). "

Reply:
I meant changes for 10+. With the way things are changing for monitoring permissions built into core in more recent versions, I don't see us needing any for the foreseeable future and these old function requirements will be gone. If we add any for 10+, we can see about splitting things up then.